src/shared/clients/defaultS3Client.ts

```diff
@@ -0,0 +1,411 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { S3 } from 'aws-sdk'
+import { ListObjectsV2Output } from 'aws-sdk/clients/s3'
```

Comment:
Can you not get this from the above import?

Reply:
You can refer to it as `S3.ListObjectsV2Output`. Changed it to clarify.